tutorials/polkadot-sdk/parachains/upgrade-to-parachain/.pages

```diff
@@ -0,0 +1,6 @@
+title: Upgrade to Parachain
```

Comment:
this sounds a little bit off

Reply:
What would you recommend? Upgrade to a Parachain? Connect to Relay Chain?

Reply:
We went with "Connect to a Relay Chain"